### ISSUE TITLE
Keyboard navigation does not work on IE11 with JAWS(screen reader)

### ISSUE THREAD

**Issue by dineshgm** [JedWatson/react-select#2509]
Hi,

With jaws screen reader on, react-select options are not selectable and the screen reader is not reading out the options.
